zebra-test/src/prelude.rs

```diff
@@ -6,5 +6,4 @@ pub use std::process::Stdio;
 pub use color_eyre;
 pub use color_eyre::eyre;
 pub use eyre::Result;
-pub use pretty_assertions::{assert_eq, assert_ne};
```

Comment:
Minor: Should we also remove this dependency from `zebra-test`?

Reply:
I removed these macros from the `zebra_test` prelude because they conflict with the `std` prelude. Which causes compiler errors whenever we `use zebra_test::prelude::*` and then use `assert_eq!`.

Since we're not using the dependency any more, we can remove it.

If we decide we want to use it in future, we should import those macros directly, to override the prelude. And add it as a dev dependency in specific crates.

Reply:
Dependency removed in 62fe70fe3